django/db/backends/dummy/base.py

```diff
@@ -83,4 +83,4 @@ def __init__(self, *args, **kwargs):
         self.validation = BaseDatabaseValidation(self)
 
     def is_usable(self):
-        return True
+        return False
```

Comment:
I added the part that checks it in makemigrations.py. Other than that, usages of is_usable on a database wrapper object seem to be only within django.db.backends and in the tests.

To me it seemed like a good place to for something that checks the status of migrations to determine whether it should continue. However, I see the problem with this now. If is_usable returns False for any real databases, then the script can't connect to the database at all. With this update, those databases would be skipped when checking the migrations, and makemigrations would complete without an error. But that's wrong -- it needs to raise an exception if it can't connect to a real database.